src/uvw/udp.hpp

```diff
@@ -278,12 +278,12 @@ class UDPHandle final: public Handle<UDPHandle, uv_udp_t> {
 
     /**
      * @brief Sets the multicast interface to send or receive data on.
-     * @param interface Interface address.
+     * @param interface_addr Interface address.
```

Comment:
`uvw` coding style comprises camel case, `libuv` uses underscores. I would suggest `iface` instead of `interface_addr`.